cluster/demo/Main.hs

```diff
@@ -73,6 +73,7 @@ main = void $ do
 
     handles <- forM cluster $ \case
         RunningCoreNode (NodeName nodeId) env handle -> do
+            -- todo, dont mapm over chars
```

Comment:
Sorry, I do not understand that comment.

Reply:
`putStrLn` does that for `String`, but i suspect `putTextLn` does something smarter.

Reply:
i added the comment after observing that kind of interleaving on the console output